scripts/powershell/Office-365/README.md

```diff
@@ -0,0 +1,29 @@
+# Office 365 Scripts

```

Comment:
@mattco593 , let's put this closer to the script, make a new "group-creators" folder under Office-365? Or add this as a block comment?